src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -280,12 +280,13 @@
 		</template>
 
 		<template #footer>
-			<div id="app-settings">
-				<div id="app-settings-header">
-					<NcButton class="settings-button" @click="showSettings">
-						{{ t('spreed', 'Talk settings') }}
-					</NcButton>
-				</div>
+			<div class="left-sidebar__settings-button-container">
+				<NcButton type="tertiary" wide @click="showSettings">
```

Comment:
I'm fine with current state, as it draws much less attention (and looks better :yum: ):
Talk | Mail | Calendar
-- | -- | --
![image](https://github.com/nextcloud/spreed/assets/93392545/24093f71-4132-409b-bbac-8c75a641bb21) | ![image](https://github.com/nextcloud/spreed/assets/93392545/72dce30f-2e96-42c9-ab3e-93df5d9b255e) | ![image](https://github.com/nextcloud/spreed/assets/93392545/46d753d9-2304-46f0-abae-0fdd89b56c11)

Reply:
Should this be handled in/via https://github.com/nextcloud-libraries/nextcloud-vue/blob/master/src/components/NcAppNavigationSettings/NcAppNavigationSettings.vue or a new component, to avoid all the custom code/handling in X apps?

Reply:
> Should this be handled in/via https://github.com/nextcloud-libraries/nextcloud-vue/blob/master/src/components/NcAppNavigationSettings/NcAppNavigationSettings.vue or a new component, to avoid all the custom code/handling in X apps?

I though about it but I'm not sure about it.

This cannot be handled via `NcAppNavigationSettings` (at least without making it a 2-in-1 component), because `NcAppNavigationSettings` is about "deprecated" collapsible settings, not a button.

This could be a new component, but its usage would probably have more variability than commons:
- In some apps, `AppNavigation` is fulfilled with navigation items and the settings button should look like a navigation item (Files, Photos)
- In some apps it would be secondary (Mail)
- In some apps it would be tertiary (Talk)

The only common is that it is a wide `NcButton`.

But we can discuss it with designers in `@nextcloud/vue`

Reply:
> I'm fine with current state, as it draws much less attention (and looks better 😋 ):

The current state - in `main` or in this PR (tertiary)?

Reply:
> In some apps it would be secondary (Mail)
In some apps it would be tertiary (Talk)

This is exactly why I think it should be a common component.
I'm not sure the difference is intentional.

Reply:
Talk and Mail use simple NcButton as a trigger for opening a dialog
Calendar has special component with "accordion" feature: https://github.com/nextcloud-libraries/nextcloud-vue/blob/master/src/components/NcAppNavigationSettings/NcAppNavigationSettings.vue

So I'd say, it would be better to summon design team here, to come to a conclusion on all three apps, then apply resulting styles on Mail and Vue-lib for Calendar

Reply:
cc @marcoambrosini

Reply:
The accordion should go asap, in favor of the settings dialog with proper explanations of the several settings.
I like both the secondary y and tertiary versions